docs/develop/api-reference/index.mdx

```diff
@@ -43,7 +43,7 @@ Attributes:
 
 ### List caches
 
-Lists all caches.
+Lists all caches for the provided auth token.
```

Comment:
@NoSQLKnowHow I think we should fix this again in another follow up pr.

Reply:
Will do